float64 rc_roll
float64 rc_pitch
float64 rc_throttle
float64 qp_roll
float64 qp_pitch
float64 qp_throttle
